src/components/manage/Blocks/Search/components/SortOn.jsx

```diff
@@ -73,6 +73,7 @@ const SortOn = (props) => {
           placeholder={intl.formatMessage(messages.sortOn)}
           styles={sortOnSelectStyles}
           theme={selectTheme}
+          label={'Sort by Dropdown'}
```

Comment:
This must be internationalized. See recent comment at https://github.com/plone/volto/issues/5400#issuecomment-1826609113